app/gui2/src/stores/project/computedValueRegistry.ts

```diff
@@ -43,7 +54,9 @@ export class ComputedValueRegistry {
   processUpdates(updates: ExpressionUpdate[]) {
     for (const update of updates) {
       const info = this.db.get(update.expressionId)
-      this.db.set(update.expressionId, combineInfo(info, update))
+      const newInfo = combineInfo(info, update)
+      this.emit('update', [update, info, newInfo])
```

Comment:
Is this emit still used? If not, let's remove the whole `ObservableV2` from here, as it's no longer necessary.

Reply:
fixed.